common/test/acceptance/pages/studio/html_component_editor.py

```diff
@@ -7,25 +7,40 @@ class HtmlComponentEditorView(ComponentEditorView):
     """
     Represents the rendered view of an HTML component editor.
     """
-    def set_content_and_save(self, content):
+    def set_content_and_save(self, content, raw=False):
         """
         Types content into the html component and presses Save.
         """
-        self.set_content(content)
+        self.set_content(content, raw)
         self.save()
 
-    def set_content_and_cancel(self, content):
+    def set_content_and_cancel(self, content, raw=False):
         """
         Types content into the html component and presses Cancel to abort the change.
         """
-        self.set_content(content)
+        self.set_content(content, raw)
         self.cancel()
 
-    def set_content(self, content):
+    def set_content(self, content, raw=False):
         """
         Types content into the html component, leaving the component open.
+        Optionally edits in 'raw HTML' mode as indicated by raw param.
```

Comment:
Nit: I'd use the Google-style docstring for the arguments:

https://github.com/edx/edx-platform/wiki/Python-Guidelines

    """Types content into the html component, leaving the component open.

    Arguments:
        content (str): The content to be used.
        raw (bool): If true, edits in 'raw HTML' mode.
    """